lib/msal-browser/test/interaction_client/RedirectClient.spec.ts

```diff
@@ -2740,7 +2747,7 @@ describe("RedirectClient", () => {
                 await redirectClient.acquireToken(emptyRequest);
             } catch (e) {
                 // Test that error was cached for telemetry purposes and then thrown
-                expect(window.sessionStorage).toHaveLength(2);
+                expect(window.sessionStorage).toHaveLength(1);
```

Comment:
This (and line 3335 below) are reverting a change that was wrongly made in the redirect retry PR.